server/service/desktop_client.go

```diff
@@ -0,0 +1,59 @@
+package service
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+)
+
+// Device client is used consume the `/api/_version_/fleet/device/{token}/desktop` endpoint,
+// and meant to be used by Fleet Desktop
+type DesktopClient struct {
```

Comment:
with token rotation we'll also be using this client in Orbit, I'd vote to leave it at `DeviceClient`